src/dailies.ts

```diff
@@ -591,3 +600,45 @@ function pantogram(): void {
   visitUrl("inv_use.php?pwd&whichitem=9573");
   visitUrl(`choice.php?whichchoice=1270&pwd&option=1&m=${m}&e=5&s1=5789,1&s2=706,1&s3=24,1`);
 }
+
+function pickCargoPocket(): void {
+  if (!have($item`Cargo Cultist Shorts`) || get("_cargoPocketEmptied")) return;
+
+  const picked = pickedPockets();
+  const items = itemPockets();
+  const meats = meatPockets();
+  const scraps = scrapPockets();
+
+  function pocketValue(pocket: number): number {
+    let value = 0;
+    if (pocket in picked) {
+      return value;
+    }
+    if (pocket in items) {
+      value += Object.entries(pocketItems(pocket))
+        .map(([item, count]) => getSaleValue(toItem(item)) * count)
+        .reduce((prev, cur) => prev + cur, 0);
+    }
+    if (pocket in meats) {
+      value += Object.values(pocketMeat(pocket))
+        .map((x) => parseInt(x))
+        .reduce((prev, cur) => prev + cur, 0);
+    }
+    if (pocket in scraps) {
+      value += 200;
+    }
+    return value;
```

Comment:
I don't particularly understand why you've set up this function this way. Are there pockets that fit more than one category?